src/webgl/light.js

```diff
@@ -315,4 +315,35 @@ p5.prototype.lights = function() {
   return this;
 };
 
+/**
+ * @method fallOff
+ * @chainable
+ */
+p5.prototype.fallOff = function(constAtt, linearAtt, quadAtt) {
```

Comment:
Can we name these variables `constantAttenuation, linearAttenuation, quadraticAttenuation` (and also their same properties attached to the renderer object)? I know they're a little on the long side but I think will improve readability and remove any confusion about their purpose especially with `const`.